Sources/Extensions/UIKit/UIApplication+Povio.swift

```diff
@@ -14,6 +14,12 @@ public extension UIApplication {
     URL(string: UIApplication.openSettingsURLString).map(openUrl)
   }
   
+  /// Opens `App Store` and deep linking to the app with provided id.
+  func openAppStore(appName: String, appleAppId: String) {
+    guard let url = URL(string: "itms-apps://apps.apple.com/us/app/\(appName)/id\(appleAppId)") else { return }
+    open(url, options: [:], completionHandler: nil)
+  }
```

Comment:
I think `appName` is not required. You can access app store directly by this url:
`itms-apps://apps.apple.com/us/app/id\(appleAppId)` where `appleAppId` only contains numbers (no letters, e.g. `id` prefix).